packages/block-editor/src/components/inner-blocks/index.js

```diff
@@ -106,34 +105,22 @@ class InnerBlocks extends Component {
 			clientId,
 			hasOverlay,
 			renderAppender,
-			template,
 			__experimentalMoverDirection: moverDirection,
-			__experimentalTemplateOptions: templateOptions,
-			__experimentalOnSelectTemplateOption: onSelectTemplateOption,
-			__experimentalAllowTemplateOptionSkip: allowTemplateOptionSkip,
 		} = this.props;
 		const { templateInProcess } = this.state;
 
-		const isPlaceholder = template === null && !! templateOptions;
-
 		const classes = classnames( 'editor-inner-blocks block-editor-inner-blocks', {
-			'has-overlay': isSmallScreen && ( hasOverlay && ! isPlaceholder ), // Temporary click-through disable on desktop.
+			'has-overlay': isSmallScreen && hasOverlay, // Temporary click-through disable on desktop.
 		} );
 
 		return (
 			<div className={ classes }>
 				{ ! templateInProcess && (
-					isPlaceholder ?
-						<TemplatePicker
```

Comment:
Should we remove this functionality as part of this PR? It's marked as experimental but it looks like it might break CoBlocks. See related comment: https://github.com/WordPress/gutenberg/issues/18738#issuecomment-559419550.

Reply:
If we're not able to remove experimental APIs, there's no reason to add experimental APIs anymore, just mark them stable from the start.

It is clearly defined in our guidelines that plugin authors should not rely on experimental APIs and they are there for testing purpose.

https://developer.wordpress.org/block-editor/contributors/develop/coding-guidelines/#experimental-and-unstable-apis

So yes, we should remove it.